### ISSUE TITLE
Extend Readme

### ISSUE THREAD

**Issue by tomVeloso** [pkgw/bibtools#18]
Add some lines on the readme with some simple examples of usage that underline the main features of the package.

**Comment by pkgw** [comment]
Thanks for your interest! Yes, the README should be better. I'm quite busy these days so it will likely take me a while to get to this one, but I did want to acknowledge it.

**Comment by tomVeloso** [comment]
Great! ... In the meanwhile, do you have any simple command line example to share with me? I'd love to start to try it ..

**Comment by cefect** [comment]
agreed. I can't find any documentation or an example of how to run it. spending 30 seconds to add an example CLI call would make the tool usable.
